dropwizard-e2e/src/test/java/com/example/sslreload/SslReloadAppTest.java

```diff
@@ -57,7 +58,7 @@ public void checkServerTrusted(X509Certificate[] x509Certificates, String s) {
     @BeforeAll
     public static void setupClass(@TempDir Path tempDir) throws IOException {
         keystore = tempDir.resolve("keystore.jks");
-        try (InputStream inputStream = SslReloadAppTest.class.getResourceAsStream("/sslreload/keystore.jks")) {
+        try (InputStream inputStream = requireNonNull(SslReloadAppTest.class.getResourceAsStream("/sslreload/keystore.jks"))) {
```

Comment:
@rhowe Not sure what this exactly accomplishes. If `SslReloadAppTest.class.getResourceAsStream(...)` returns `null`, this will already fail with a `NullPointerException` which is exactly what happens when wrapping it in `Objects.requireNonNull(...)`.